modules/gapi/include/opencv2/gapi/cpu/gcpukernel.hpp

```diff
@@ -224,7 +222,6 @@ void postprocess(Outputs&... outs)
     struct
     {
         void operator()(tracked_cv_mat* bm) { bm->validate();  }
-        void operator()(scalar_wrapper* sw) { sw->writeBack(); }
```

Comment:
...and it seems you don't use it already :)

Reply:
Removed `scalar_wrapper` struct. Fixed.